projects/p04-ops-monitoring/Makefile

```
@@ -17,6 +17,31 @@ run: ## Start monitoring stack
 stop: ## Stop monitoring stack
 	docker-compose down
 
+restart: stop run ## Restart monitoring stack
```

checkmake (0.2.2)

[warning] Target "restart" should be declared PHONY.

(phonydeclared)